src/components/pushClient/Banner.jsx

```diff
@@ -53,9 +54,13 @@ class BannerClient extends Component {
     track(element)
   }
 
+  isDisabled = () =>
+    Config.promoteDesktop.isActivated !== true ||
+    !this.state.mustShow ||
+    isFlagshipApp()
+
```

Comment:
We also have https://github.com/cozy/cozy-drive/blob/master/src/components/pushClient/Button.jsx

I think the right fix should be in Main.jsx https://github.com/cozy/cozy-drive/blob/master/src/drive/web/modules/layout/Main.jsx

Also, I'm wondering if we don't have this pushClient in Photo too. Can you take a look?

Reply:
will look

Reply:
The conditionals look a bit wild